README.md

```diff
@@ -266,6 +266,31 @@ end
 *   `:port` (Fixnum) - The port which should be used to communicate
     with the PhantomJS process. Defaults to a random open port.
 
+### URL Blacklisting ###
+
+The latest version of Poltergeist supports URL blacklisting, which allows you to prevent scripts from running on designated domains. If you are experiencing slower run times, consider creating a URL blacklist of domains that are not essential to your testing environment, such as ad networks or analytics.
+
+To utilize URL blacklisting, modify your Gemfile to use the latest version of Poltergeist:
+
+``` ruby
+gem 'poltergeist', git: 'https://github.com/teampoltergeist/poltergeist.git', branch: 'master'
```

Comment:
This is somewhat dangerous as a directive, since master isn't always stable; what's blocking doing a release that includes url blacklisting? Should we set up a milestone for it? Can it be backported to our latest stable release as a stand-alone feature release?